queries.sql

```diff
@@ -138,3 +138,68 @@ SELECT owners.full_name, count(*) FROM animals
 JOIN owners on animals.owner_id = owners.id
 GROUP BY owners.full_name
 ORDER BY count DESC LIMIT 1;
+
+/* Project 4 */
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name, visits.date_of_visit FROM animals
+INNER JOIN visits on animals.id = visits.animals_id
+INNER JOIN vets on vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER by date_of_visit DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT count(*) as differnt_animals FROM animals
+INNER JOIN visits on animals.id = visits.animals_id
+INNER JOIN vets on vets.id = visits.vet_id
+WHERE vets.name = 'Stephanie Mendez';
```

Comment:
- [ ] Kindly be noticed that you should use the `DISTINCT` keyword in this query.

```sql
  SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```